front/src/pages/AdminPage.tsx

```diff
@@ -0,0 +1,50 @@
+import AdminPanelNavbar from "../components/AdminPanelNavbar.tsx";
+import AdminPageSideBar from "../components/AdminPageSideBar.tsx";
+import {useParams} from "react-router-dom";
+import Overview from "./AdminPages/Overview.tsx";
+import Scheduling from "./AdminPages/Scheduling.tsx";
+import Vacations from "./AdminPages/Vacations.tsx";
+import Employees from "./AdminPages/Employees.tsx";
+
+
+const AdminPage = () => {
+    const {'*': key} = useParams();
+
+    let innerComponent;
+
+    switch (key) {
+        case 'overview':
```

Comment:
declare all the options in another file. and reference these here and in the navbar